.gitmodules

```diff
@@ -46,10 +46,6 @@
 
 # PMM Server
 
-[submodule "pmm-managed"]
```

Comment:
I think we need to do much more to remove a submodule ([link](https://git.wiki.kernel.org/index.php/GitSubmoduleTutorial#Removal))

Reply:
Done. Also we need to do the same for pmm-admin and pmm-agent?